src/use/use.py

```diff
@@ -455,7 +456,8 @@ def find_latest_working_version(releases: Dict[str, List[Dict[str, str]]], # {ve
             if is_version_satisfied(info, sys_version) and \
                 is_platform_satisfied(info, platform_tags) and \
                 is_interpreter_satisfied(info, interpreter_tag):
-                return info["version"], info["hash"]
+                hash_value = info["digests"].get(hash_algo.name)
```

Comment:
might need to reconsider this step and raise an exception instead if no hash is available

Reply:
Addressed in commit #3